packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -1224,13 +1224,13 @@ class _RenderDecoration extends RenderBox with SlottedContainerRenderObjectMixin
   @override
   double computeMinIntrinsicWidth(double height) {
     return _minWidth(icon, height)
-      + contentPadding.left
+      + (prefixIcon == null ? contentPadding.left : 0)
```

Comment:
This doesn't look right when the writing direction is RTL?

Reply:
Yeah you are correct. I have fixed this.